test/has.js

```diff
@@ -22,4 +22,15 @@ describe('has', function() {
     eq(R.has('age', bob), false);
   });
 
+  it('tests paths on non-objects', function() {
```

Comment:
I'd like to find better phrasing here, too, although it's harder to come up with something.